language/stdlib/modules/LibraSystem.move

```diff
@@ -262,6 +272,11 @@ module LibraSystem {
         let to_update_index_vec = get_validator_index_(&libra_system_config.validators, validator_addr);
         assert(Option::is_some(&to_update_index_vec), Errors::invalid_argument(ENOT_AN_ACTIVE_VALIDATOR));
         let to_update_index = *Option::borrow(&to_update_index_vec);
+
+        assert(LibraTimestamp::now_microseconds() >
```

Comment:
Should this go inside the branch at 281?

I think the only difference in behavior is that a "no-op" config update within 5 minutes of the last config update will 
1. abort as the code is currently written
2. succeed, but do nothing if we put this inside the branch

Reply:
Indeed, would makes sense to succeed if nothing has changed even if the rate limit is not met.